AGPs/AGP: Nest 2019 Budget.md

```diff
@@ -0,0 +1,38 @@
+---
+AGP: N/A
+Title: Nest 2019 Budget 
+Author: Maria Gomez (@mariapao)
+Status: Stage III
+Track: Finance
+Created: 2018-01-09
+---
+
+## Address of the transfer recipient
+
+The funding will be managed by the Nest DAO and therefore, we request the funds to be sent to the mentioned DAO. The URL of the Nest DAO where the funds can be sent will be provided once the DAO is created.
+
+## Amount of the transfer
+
+$500k in DAI (or other stable assets, such as fiat)
```

Comment:
It may make sense to remove this, particularly the fiat part as fiat cannot currently be sent or managed by the DAO.

Reply:
You're right. Done :)